lib/Teepee.js

```diff
@@ -148,6 +148,12 @@ Teepee.prototype.subsidiary = function(config) {
     that.emit.apply(that, arguments);
   };
 
+  const subsidiaryOn = subsidiary.on;
```

Comment:
This ensures that the event handler will be added to both subsidiary instance and the base client instance.
We needed this because we are emitting events to both instances.

Reply:
Can you write a few words about how/why you're emitting errors to both instances?

I have a feeling that it might be better to only emit to the base if there are no `error` listeners on the subsidiary 🤔

Reply:
Hmm... `error` events bubbling up indeed seems a bit... surprising. 

Isn't this just a matter of missing errorhandling at the request size? I don't think the subsidiary clients that we generally use across apps rely on centralized error handling?

Reply:
Your test says that it `should not throw unhandled error when emitting error from subsidiary object`. I understand that as you used to get an unhandled error exception when emitting on the subsidiary client. Isn't that an indication that we should "scope" the emit method properly, and not have events go between the parent and the subsidiary, and NOT an argument for attaching listeners to both the subsidiary and the parent?

With this change, your spy passed as the handler for the `.on('event')` call on the subsidiary will be called twice - won't it?

Reply:
@papandreou in line 145 we are emitting events on both instances.

  const that = this;
  const subsidiaryEmit = subsidiary.emit;
  subsidiary.emit = function() {
    subsidiaryEmit.apply(this, arguments);
    that.emit.apply(that, arguments);
  };
  
So we need a way to attach event handlers to the base client when we are adding event handlers via subsidiary instance.

@gustavnikolaj we have request error handler but we are still encountering unhandled error.

Reply:
I think the proper fix here should be to scope the emit method to only emit the event on the subsidiary.

The parent shouldn't receive `error` events emitted on the child.

Reply:
With the caveat, as @papandreou says, it make sense to emit it on the parent, in the specific case where there is no listener for errors on the child.

Reply:
@gustavnikolaj @papandreou i'll make the change to check for error handler, If no error handler exist in subsidiary then parent should emit error event.

Reply:
@outlawtorn, ah right, then I just misunderstood who "we" were in your above statement 🙈

Reply:
@papandreou @gustavnikolaj i updated the fix, can you please check? Thank you.